lua/entities/gmod_wire_expression2/core/custom/tracesystem.lua

```diff
@@ -1349,7 +1316,8 @@ e2function vector rayCircleIntersection( vector start, vector dir, vector pos, v
 end
 
 e2function vector raySphereIntersection( vector start, vector dir, vector pos, number radius )
-	return RaySphereIntersection( start, dir, pos, radius )
+	local P = RaySphereIntersection( start, dir, pos, radius )
+  return (P ~= nil) and P or Vector()

```

Comment:
Just `P or Vector()` should work. Or just make RaySphereIntersection return Vector() instead of nil.

Reply:
Fixed in https://github.com/wiremod/wire-extras/pull/122/commits/eaf547b2b4d89be675565047956acdc118204027